org.pwsafe.pwsafe.yml

```diff
@@ -0,0 +1,65 @@
+app-id: org.pwsafe.pwsafe
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: pwsafe
+rename-icon: pwsafe
+rename-desktop-file: pwsafe.desktop
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=ipc
+  - --filesystem=xdg-documents
+  - --env=PWS_HELPDIR=/app/share/passwordsafe/help/
+  # YubiKey USB access
+  - --device=all
+cleanup-commands:
```

Comment:
I am user of Password Safe program and few months ago I have created manifest file, build flatpak successfully, zipped all the files required and uploaded them to [application developer github issue tracker site](https://github.com/pwsafe/pwsafe/issues/562) and finally communicated with developer, he will submit the flatpak. Developer @ronys was busy and now after few months reported submitted manifest...

I have only built one single flatpak, this one, and like I remember there were two commands to clean up. First one was for deleting files and I was struggling with it to delete whole subdirectories, so that is the reason I used "cleanup-commands" option, to just use "rm" Linux command to delete all required subdirectories. Any help how to clean those subdirectories is much appreciated.

Reply:
https://docs.flatpak.org/en/latest/manifests.html#cleanup